tests/storage/snapshots/conftest.py

```
@@ -149,3 +173,202 @@ def vm_restore_with_predictable_names(
+                namespace_name=namespace.name,
+                client=unprivileged_client,
+                storage_class_name=snapshot_storage_class_name_scope_module,
+                data_source=fedora_data_source_scope_module,
+            )
+            vms.append(vm)
+        yield vms
+    finally:
+        for vm in vms:
+            try:
+                vm.clean_up()
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)